### ISSUE TITLE
Not working with Cura 5.0

### ISSUE THREAD

**Issue by ungutknut** [Ultimaker/cura-siemensnx-plugin#6]
Cura 5.0 has a new folder structure and also the registry keys are located somewhere else. Therefore the latest cura version this plugin works with is arachne beta 2 (surprisingly the plugin also works with NX13 BTW).

I located the new path and it might work now but I can't test it since I can't get the installer to run (I'm not a programmer... barely know what I'm doing here). 

Anyway I edited the .vb file according to the new reg key location. 

Maybe someone could test this and merge it with the marketplace version of this plugin:
[NXCuraIntegrationCleanUI.zip](https://github.com/Ultimaker/cura-siemensnx-plugin/files/8727744/NXCuraIntegrationCleanUI.zip)

**Comment by paprott8s** [comment]
@ungutknut Yep, running a similar issue. Updated to Cura 5.0 and I am really suffering from the lack of this integration. My previous version of Cura was 4.6.1 and was still running fine with NX 20.00

**Comment by derrapf** [comment]
Mee too.
Would appreciate if some one could fix it
